product_code_builder_sequence/models/product_product.py

```diff
@@ -53,11 +53,11 @@ class ProductProduct(models.Model):
 
     @api.multi
     def copy(self, default=None):
-        for product in self:
-            if default is None:
-                default = {}
-            if product.prefix_code:
-                default.update({
-                    'prefix_code': self.prefix_code + _('-copy'),
-                })
+        self.ensure_one()
```

Comment:
i'm not thinking about the return, so maybe in this case it's ok to use ensure_one so it's :+1:  for me